src/ert/config/queue_config.py

```diff
@@ -18,6 +18,7 @@
 )
 
 GENERIC_QUEUE_OPTIONS: List[str] = ["MAX_RUNNING"]
+PROJECT_CODE = ["PROJECT_CODE"]
```

Comment:
Why do we need a new category named the same as its only value?

Reply:
At first, I wanted to add it to `GENERIC_QUEUE_OPTIONS` but I did not want it to be an option for `VALID_QUEUE_OPTIONS-QueueSystem.LOCAL` so this was what I came up with. This will probably be refactored as part of https://github.com/equinor/ert/issues/8201. As this requires a list of keys that will be valid for all drivers and this PROJECT_CODE falls under this category.

Reply:
I think it is cleaner to inject it inside each `*_DRIVER_OPTIONS`  then, or we should have a category for things valid for all drivers except the LocalDriver.